ArtOfIllusion/src/artofillusion/ui/UIUtilities.java

```diff
@@ -251,4 +253,37 @@ private static void addChildrenToList(Widget w, List<Widget> list)
         addChildrenToList(child, list);
       }
   }
+
+  /**
+   * Test event for specified mouse/modifier key combination.
+   * This regularizes 3-button and 1-button + modifier support across
+   * multiple platforms.
+   */
+
+  public static boolean mouseButtonOne(WidgetMouseEvent e)
+  {
+    int mods = e.getModifiersEx();
+    return ((mods & BUTTON1_DOWN_MASK) != 0) &&
+           ((mods & (CTRL_DOWN_MASK | META_DOWN_MASK | ALT_DOWN_MASK)) == 0);
```

Comment:
The this would prevent usin Ctrl as modifier (as for example picking individual objects on screen or removing them from selection).

```suggestion
            ((mods & (META_DOWN_MASK | ALT_DOWN_MASK)) == 0);
```

Reply:
Good catch.